README.md

```diff
@@ -62,7 +62,7 @@ If you want bread at lunchtime, begin around 8/9am.
 
 ![petrissage_2][petrissage_2]
 
-* Knead the sticky dough during 10 mins (if you have a Thermomix: 12mn / 37 deg / Mess mode)
+* Knead the sticky dough during 10 mins (if you have a Thermomix: 12mn / 37 deg / Mess mode). It is sticky because the dough contains much water: it is normal, baker's yeast microorganisms need water to develop, it is the insurance to have a airy bread. I slightly oil my hand and fold the dough along itself during 10 mn.
```

Comment:
```suggestion
* Knead the sticky dough for 10 mins (if you have a Thermomix: 12mn / 37 deg / Mess mode). The dough is sticky because it contains quite a bit of water. This is expected, as baker's yeast microorganisms, which make for an airy bread, need water to develop.
If you are kneading by hand, I recommend oiling your hand and folding dough on itself for 10 mins.
```